lib/NetPacks.h

```diff
@@ -465,7 +465,7 @@ struct SetCommanderProperty : public CPackForClient //120
 	ECommanderProperty which;
 	TExpType amount; //0 for dead, >0 for alive
 	si32 additionalInfo; //for secondary skills choice
-	Bonus accumulatedBonus;
+	std::shared_ptr<Bonus> accumulatedBonus;

```

Comment:
I`m afraid this is also wrong (this object is not constructed anywhere). Also I`m not sure that shared_ptr works correctly with netpacks.